nav2_amcl/src/amcl_node.cpp

```diff
@@ -629,6 +629,13 @@ void AmclNode::updatePoseFromServer()
 void
 AmclNode::checkLaserReceived()
```

Comment:
This is interesting in that it is almost a periodic self-test. In this case to confirm that it is receiving its expected inputs. We may want to consider generalizing this idea and applying it in a uniform way to all of the modules.